src/WFQC/data.py

```diff
@@ -35,61 +36,61 @@ async def aggregate_requests(session, url):
     
     async with session.get(url) as response:
         return await response.json()
-    
-async def get_biotools_metadata(outpath, topicID="topic_0121"):  # TODO: I removed format. Check if there is any reason to have it 
-                                                        # TODO: should add parameter for optional forced retrieval - even if csv file, still recreate it 
-                                                        # TODO: Currently no timing - add tracker
-    """
-    Fetches metadata about tools from bio.tools, belonging to a given topicID and returns as a dataframe.
-    If a CSV file already exists load the dataframe from it. 
 
-    Parameters
-    ----------
-    topicID : str TODO: make this a int instead? why am I writing topic? 
-        The ID to which the tools belongs to, ex. "Proteomics" or "DNA" as defined by 
-        EDAM ontology (visualisation: https://edamontology.github.io/edam-browser/#topic_0003)
+async def get_pmid_from_doi(doi_tools):
```

Comment:
A description of the method would be useful here.

Reply:
In general it's good to document all the methods, I see that you were documenting the ones you deleted, but just a reminder. The numpy style you were using is good for that 👍